datalab/datalab_session/util.py

```diff
@@ -141,23 +149,25 @@ def stack_arrays(array_list: list):
 
   return stacked
 
-def scale_flip_points(small_img_width: int, small_img_height: int, img_array: list, points: list[tuple[int, int]]):
+def scale_points(height_1: int, width_1: int, height_2: int, width_2: int, x_points=[], y_points=[], flip_y = False, flip_x = False):
   """
-    Scale the coordinates from a smaller image to the full sized fits so we know the positions of the coords on the 2dnumpy array
-    Returns the list of tuple points with coords scaled for the numpy array
+    Scales x_points and y_points from img_1 height and width to img_2 height and width
+    Optionally flips the points on the x or y axis
   """
-  large_height, large_width = np.shape(img_array)
+  if height_1 == 0 or width_1 == 0 or height_2 == 0 or width_2 == 0:
```

Comment:
you can write this a bit more cleanly as 

```
if any([dim == 0 for dim in [height_1, width_1, height_2, width_2]]):
    raise ValueError("height and width must be non-zero")
```
`any()` returns true if any of the values in the iterable are True :) 

List comprehension: https://docs.python.org/3/tutorial/datastructures.html#list-comprehensions